lib/plugins/cors.js

```diff
@@ -80,15 +89,22 @@ function matchOrigin(req, origins) {
 function cors(opts) {
     assert.optionalObject(opts, 'options');
     opts = opts || {};
-    assert.optionalArrayOfString(opts.origins, 'options.origins');
     assert.optionalBool(opts.credentials, 'options.credentials');
     assert.optionalArrayOfString(opts.headers, 'options.headers');
 
+    if (opts.origins) {
+        assert.ok(Array.isArray(opts.origins), 'options.credentials');
```

Comment:
I think the string here should be 'options.origins'. Also, any reason not to use `assert.array`?